classlib/Employee.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
+ * Employee Class
+ *.
+ * @package
+ * @subpackage
+ * @category
+ * @author Sanjeeb Rao
+ */
+class Employee {
+
+    private $signup_fields = array(
+        'email' => '',
+        'password' => '',
+        'is_completed' => '',
+    );
+
+    private $form_fields = array(
+        'is_completed' => '1',
+        'prefix' => '',
+        'first_name' => '',
+        'middle_name' => '' ,
+        'last_name' => '',
+        'gender' => '',
+        'date_of_birth' => '',
+        'marital' => '',
+        'employment' => '',
+        'employer' => '',
+        'note' => '',
+        'communication' => '',
+        'photo' => ''
+    );
+
+    private $table_name = 'employee';
+    private $db_obj = NULL;
+
+    /**
+     * Initialization of Database object.
+     *
+     * @access public constructor of Employee
+     * @param  object db_object
+     */
+    public function __construct($db_object)
+    {
+        $this->db_obj = $db_object;
+    }
+
+    /**
+     * Create account of user.
+     *
+     * @access public create_account
+     * @param  string email
+     * @param  string password
+     * @return mixed
+     */
+    public function create_account($email, $password)
+    {
+        $this->signup_fields['email'] = $email;
+        $this->signup_fields['password'] = $password;
+        $this->signup_fields['is_completed'] = '0';
+
+        if ($this->db_obj->insert($this->table_name, $this->signup_fields) === TRUE)
+        {
+            return $this->db_obj->get_last_insert_id();
+        }
+
+        return FALSE;
+    }
+
+    /**
+     * Get employee data from database.
+     *
+     * @access public  get_employee
+     * @param  integer employee_id
+     * @return mixed
+     */
+    public function get_employee($employee_id = 0)
+    {
+        $value = 'employee.is_completed AS is_completed, employee.id AS emp_id,
+            employee.first_name AS first_name, employee.middle_name AS middle_name,
+            employee.last_name AS last_name, employee.date_of_birth AS date_of_birth,
```

Comment:
use short alias like emp instead of using employee table name prefix with column names for shorter code